identity/src/main/java/com/stripe/android/identity/analytics/IdentityAnalyticsRequestFactory.kt

```diff
@@ -4,17 +4,21 @@ import android.content.Context
 import com.stripe.android.core.networking.AnalyticsRequestV2
 import com.stripe.android.core.networking.AnalyticsRequestV2Factory
 import com.stripe.android.identity.IdentityVerificationSheetContract
+import com.stripe.android.identity.injection.IdentityVerificationScope
 import com.stripe.android.identity.networking.models.DocumentUploadParam
+import com.stripe.android.identity.networking.models.VerificationPage
 import com.stripe.android.identity.states.IdentityScanState
 import javax.inject.Inject
 
 /**
  * Factory for creating [AnalyticsRequestV2] for Identity.
  */
+@IdentityVerificationScope
 internal class IdentityAnalyticsRequestFactory @Inject constructor(
     context: Context,
     private val args: IdentityVerificationSheetContract.Args
 ) {
+    lateinit var verificationPage: VerificationPage
```

Comment:
this is `lateinit` as `IdentityAnalyticsRequestFacotry` is initialized during dagger, but `verifcationPage` can only be retrieved through a network call after